fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/EcamMessages/AbnormalSensed/ata26.tsx

```diff
@@ -16,4 +16,2039 @@ import { AbnormalProcedure } from 'instruments/src/MsfsAvionicsCommon/EcamMessag
 //    9 for ABN non-sensed procedures
 
 /** All abnormal sensed procedures (alerts, via ECL) should be here. */
-export const EcamAbnormalSensedAta26: { [n: number]: AbnormalProcedure } = {};
+export const EcamAbnormalSensedAta26: { [n: number]: AbnormalProcedure } = {
+  260800001: {
+    title: '\x1b<2m\x1b4mAPU FIRE',
+    sensed: true,
+    items: [
+      {
+        name: 'APU FIRE P/B',
+        sensed: true,
+        labelNotCompleted: 'PUSH',
+      },
+      {
+        name: 'APU AGENT AFTER 10 S',
+        sensed: true,
+        labelNotCompleted: 'DISCH',
+      },
+      {
+        name: 'APU MASTER SW',
+        sensed: true,
+        labelNotCompleted: 'OFF',
+      },
+    ],
+    recommendation: 'LAND ASAP',
+  },
+  260800002: {
+    title: '\x1b<4m\x1b4mAPU FIRE\x1bm DET FAULT',
+    sensed: true,
+    items: [],
+  },
+  260800003: {
+    title: '\x1b<4m\x1b4mAPU FIRE\x1bm LOOP A FAULT',
+    sensed: true,
+    items: [],
+  },
+  260800004: {
+    title: '\x1b<4m\x1b4mAPU FIRE\x1bm LOOP B FAULT',
+    sensed: true,
+    items: [],
+  },
+  260800005: {
+    title: '\x1b<2m\x1b4mENG 1\x1bm FIRE', // IN FLIGHT
+    sensed: true,
+    items: [
+      {
+        name: 'THR LEVER 1',
+        sensed: true,
+        labelNotCompleted: 'IDLE',
+      },
+      {
+        name: 'ENG 1 MASTER',
+        sensed: true,
+        labelNotCompleted: 'OFF',
+      },
+      {
+        name: 'ENG 1 FIRE P/B',
+        sensed: true,
+        labelNotCompleted: 'PUSH',
+      },
+      {
+        name: 'APU BLEED',
+        sensed: true,
+        labelNotCompleted: 'OFF',
+        level: 1,
+      },
+      {
+        name: 'APU BLEED : DO NOT USE',
+        sensed: false,
+        style: ChecklistLineStyle.Cyan,
+        level: 1,
+      },
+      {
+        // When timer elapsed, this should change to 'AGENT 1...DISCH'
+        name: 'AGENT 1 AFTER 10 S',
+        sensed: true,
+        labelNotCompleted: 'DISCH',
+      },
+      {
+        name: 'ATC',
+        sensed: false,
+        labelNotCompleted: 'NOTIFY',
+      },
+      {
+        // When timer elapsed, this should change to 'AS STILL FIRE AFTER 30 S'
+        name: '.IF STILL FIRE AFTER 30 S :',
```

Comment:
We don't need a dot in front of the condition on the A380. Also valid for all the other conditions, should be fixed with a search & replace on ".IF"

Reply:
Ah cool, makes it simpler, fixed now! Thanks.